multiwallet.py

```diff
@@ -246,6 +246,35 @@ def _get_bip39_firstwords():
         return fw
 
 
+#####################################################################
+# Dice rolls
+#####################################################################
+
+
+def _get_num_words():
+    num_words = _get_int(
+        prompt="How many mnemonic words should be generated?",
+        default=24,
+        minimum=12,
+        maximum=24,
+    )
+    if num_words not in (12, 15, 18, 21, 24):
+        raise ValueError(
```

Comment:
This entire method needs a `while True` block, otherwise providing an invalid entry will break (good), but the UI of that for the end-user starts them over (bad).